tests/conftest.py

```diff
@@ -2343,8 +2343,9 @@ def finalizer():
 
     # Checks OCP version
     ocp_version = get_running_ocp_version()
+    ocp_version = ["stable" if ocp_version >= "4.7" else ocp_version]
```

Comment:
You can do this without the list comprehension to keep `ocp_version` a string. This way you don't need to use `ocp_version[0]` below.

```suggestion
    ocp_version = "stable" if ocp_version >= "4.7" else ocp_version
```

Reply:
Done